addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -1323,10 +1350,35 @@ MockServer.include({
      * @param {integer} message_id
      */
     _mockMailChannel_SetLastSeenMessage(ids, message_id) {
+        // deprecated: update mail.channel/fetched_message_id and mail.channel/seen_message_id
         this._mockWrite('mail.channel', [ids, {
             fetched_message_id: message_id,
             seen_message_id: message_id,
         }]);
+        const cps = this._getRecords('mail.channel.partner', [['channel_id', 'in', ids], ['partner_id', '=', this.currentPartnerId]]);
+        this._mockWrite('mail.channel.partner', [cps.map(cp => cp.id), {
+            fetched_message_id: message_id,
+            seen_message_id: message_id,
+        }]);
+    },
+    /**
+     * Simulates `_inverse_channel_partner_ids` on `mail.channel`
+     *
+     * @private
+     */
+     _mockMailChannel__inverseChannelPartnerIds(channelIds) {
```

Comment:
single _ in the middle

Reply:
The method in python is `_inverse_channel_partner_ids`, so there's an underscore before.
There is no convention to not use similar name regardless of private/public prefix with `_`, so I prefer to consider `_f` and `f` as 2 different names, to play it safe.

Reply:
Yes but non _ method have 0 not 1 (it's just camel case).

Reply:
Oh ok, I thought there was at least one `_` to separate `modelName` and `methodName`.